test-project-vite/package.json

```diff
@@ -6,21 +6,30 @@
   "author": "Paolo Caleffi <p.caleffi@dreamonkey.com>",
   "private": true,
   "scripts": {
-    "sync:testing": "cd ../packages/testing && yarn build:local && cd ../../test-project-vite && yarn add -D ../packages/testing",
-    "sync:jest": "cd ../packages/unit-jest && yarn build:local && cd ../../test-project-vite && yarn add -D ../packages/unit-jest",
-    "sync:cypress": "cd ../packages/e2e-cypress && yarn build:local && cd ../../test-project-vite && yarn add -D ../packages/e2e-cypress",
-    "sync:all": "yarn sync:testing && yarn sync:jest && yarn sync:cypress",
+    "sync:testing": "yarn --cwd ../packages/testing build:local && yarn add -D ../packages/testing",
+    "sync:vitest": "yarn --cwd ../packages/unit-vitest build:local && yarn add -D ../packages/unit-vitest",
+    "sync:jest": "yarn --cwd ../packages/unit-jest build:local && yarn add -D ../packages/unit-jest",
+    "sync:cypress": "yarn --cwd ../packages/unit-cypress build:local && yarn add -D ../packages/unit-cypress",
+    "sync:all": "yarn sync:testing && yarn sync:vitest && yarn sync:jest && yarn sync:cypress",
+    "invoke:testing": "quasar ext invoke @quasar/testing",
+    "invoke:vitest": "quasar ext invoke @quasar/testing-unit-vitest",
+    "invoke:jest": "quasar ext invoke @quasar/testing-unit-jest",
+    "invoke:cypress": "quasar ext invoke @quasar/testing-unit-cypress",
+    "invoke:all": "yarn invoke:testing && yarn invoke:vitest && yarn invoke:jest && yarn invoke:cypress",
     "lint": "eslint --ext .js,.ts,.vue ./",
     "format": "prettier --write \"**/*.{js,ts,vue,scss,html,md,json}\" --ignore-path .gitignore",
     "test": "echo \"See package.json => scripts for available tests.\" && exit 0",
-    "test:unit:ui": "majestic",
-    "test:unit": "jest --updateSnapshot",
-    "test:unit:ci": "jest --ci",
-    "test:unit:coverage": "jest --coverage",
-    "test:unit:watch": "jest --watch",
-    "test:unit:watchAll": "jest --watchAll",
-    "serve:test:coverage": "quasar serve test/jest/coverage/lcov-report/ --port 8788",
-    "concurrently:dev:jest": "concurrently \"quasar dev\" \"jest --watch\"",
+    "test:unit:vitest:ui": "vitest --ui",
```

Comment:
I differentiated Vitest and Jest commands, remember to deselect "add scripts" when doing the sync + invoke to regenerate files